src/lib/journey/callbacks/choice/choice.svelte

```diff
@@ -17,6 +19,9 @@
   const choiceOptions = callback
     .getChoices()
     ?.map((label, idx) => ({ text: label, value: `${idx}` }));
+  const choiceOptionsForRadio = callback
+    .getChoices()
+    ?.map((label, idx) => ({ text: label, value: idx }));
```

Comment:
I had to duplicate `choiceOptions` here since the Radio composition expects the choices to be formatted as `{ text: string, value: number } `as opposed to `{ text: string, value: string }` which is what Select expects. I would welcome suggestions here for how we can improve this.